source/reference/options.txt

```diff
@@ -452,15 +452,28 @@ Authentication Options
 
         - External authentication using LDAP. You can also use ``PLAIN``
           for authenticating in-database users. ``PLAIN`` transmits
-          passwords in plain text. This mechanism is available only in
-          `MongoDB Enterprise
-          <http://www.mongodb.com/products/mongodb-enterprise-advanced?jmp=docs>`_.
+          passwords in plain text. This mechanism is
+          available in `MongoDB Enterprise
+          <http://www.mongodb.com/products/mongodb-enterprise-advanced?jmp=docs>`_
+          and `MongoDB Atlas
+          <https://www.mongodb.com/atlas/database?tck=docs_server>`_.
 
       * - :manual:`MONGODB-OIDC </core/security-oidc/>` (OpenID Connect) 
 
         - External authentication using OpenID Connect. This mechanism is
-          available only in `MongoDB Enterprise
-          <http://www.mongodb.com/products/mongodb-enterprise-advanced?jmp=docs>`_.
+          available in `MongoDB Enterprise
+          <http://www.mongodb.com/products/mongodb-enterprise-advanced?jmp=docs>`_
+          and `MongoDB Atlas
+          <https://www.mongodb.com/atlas/database?tck=docs_server>`_.
+
+      * - ``MONGODB-AWS`` (AWS IAM) 
```

Comment:
(linking)
Can you work with Fuat, Anna, or Gaurab to identify the best place to link for MONGODB-AWS information. Here is the location in Atlas, for example: https://www.mongodb.com/docs/atlas/security/aws-iam-authentication/

Also, is this needed somewhere in the server or shell docs? A quick search didn't turn up an obvious link here.

Reply:
Per our Slack convo, will ask Anna to review the PR and see if and how AWS IAM and EA need to be doc'ed.

Reply:
I don't think I understand the question here. This PR should cover what is required for mongosh – I'd expect the server to also document this feature, similar to how Atlas does that.